specification/iotcentral/data-plane/readme.md

```diff
@@ -0,0 +1,67 @@
+# IoT Central - API client generation
+
+> see https://aka.ms/autorest
+
+This is the AutoRest configuration file for MonitorClient.
+
+
+
+---
+## Getting Started
+To build the SDK for MonitorClient, simply [Install AutoRest](https://aka.ms/autorest/install) and in this folder, run:
+
+> `autorest`
+
+To see additional help and options, run:
+
+> `autorest --help`
+---
+
+## Configuration
+These are the global settings for the IoT Central API.
+
+```yaml
+version: latest
+clear-output-folder: true
+
+# azure-validator: true
+semantic-validator: true
+
+azure-arm: true
+add-credentials: true
+generate-metadata: true
+license-header: MICROSOFT_MIT_NO_VERSION
+
+csharp:
+    namespace: Microsoft.Azure.IotCentral
+    output-folder: client/csharp/Microsoft.Azure.IotCentral
+
+nodejs:
+    package-name: azure-iotcentral
+    package-version: 1.0.0
+    output-folder: client/nodejs
+```
+
+## Multi-API/Profile support for AutoRest v3 generators 
+
+AutoRest V3 generators require the use of `--tag=all-api-versions` to select api files.
+
+This block is updated by an automatic script. Edits may be lost!
+
+``` yaml $(tag) == 'all-api-versions' /* autogenerated */
+# include the azure profile definitions from the standard location
+require: $(this-folder)/../../../profiles/readme.md
+
+# all the input files across all versions
+input-file:
+  - $(this-folder)/preview/2019-10-28/iotcentral.json
```

Comment:
This path is not correct, pls change it to 
`$(this-folder)/Microsoft.IoTCentral/preview/2019-10-28/iotcentral.json`

Reply:
thanks   @ruowan  this is passing now.  The only remaining issue is the breaking change validation it seems oad is failing to de-serialize the swagger to do the check , but when i run the tool locally it passes fine.